cmd/cerbosctl/internal/messages.go

```diff
@@ -0,0 +1,9 @@
+// Copyright 2021 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package internal
+
+const (
+	FeedbackMsg  = "It looks like an unexpected error happened. Would you mind reporting it to the link below?"
+	FeedbackLink = "https://github.com/cerbos/cerbos/issues/new/choose"
```

Comment:
How about generating a link that creates a specific type of issue with the appropriate title, label and body from the stacktrace? https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-a-url-query

Reply:
Sounds good but the content may be look a bit messy with the URL encoding. Nonetheless we can try it 👍